test/docker-compose.integration-tests.yml

```diff
@@ -44,112 +44,23 @@ services:
     volumes:
       - *read-only-pulumi-mnt
 
-  analyzerlib-integration-tests:
-    image: grapl/analyzerlib-test:${TAG:-latest}
+  python-integration-tests:
```

Comment:
Not gonna lie, I'm gonna be semi surprised if running all the tests against one `docker-compose up` instance will work without some tweaking. (I just assume that each integration test previously assumed clean state on the dbs, and thought it could muck with it as-needed)

Reply:
We'll have to address this if it comes up. The current plan is to use `tenant_id` to orthogonalize tests.